content/maxwell1_fundamentals/interface_conditions/index.rst

```diff
@@ -0,0 +1,48 @@
+
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|Property                         | Normal Component                                           | Tangential Component                                          |
++=================================+============================================================+===============================================================+
+|:math:`\mathbf{e}`               |:math:`\varepsilon_1e_1^\perp-\varepsilon_2e_2^\perp=\tau_f`|:math:`e_1^\parallel-e_2^\parallel=0`                          |
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|:math:`\mathbf{d}`               | :math:`d_1^\perp-d_2^\perp=\tau_f`                         |:math:`\varepsilon_2d_1^\parallel-\varepsilon_1d_2^\parallel=0`|
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|:math:`\mathbf{j}` (steady-state)| :math:`j_1^\perp=j_2^\perp`                                |:math:`\rho_1j_1^\parallel-\rho_2 j_2^\parallel=0`             |
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|:math:`\mathbf{h}`               |:math:`\mu_1h_1^\perp-\mu_2h_2^\perp=0`                     |:math:`h_1^\parallel-h_2^\parallel=K_f`                        |
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|:math:`\mathbf{b}`               | :math:`b_1^\perp-b_2^\perp=0`                              |:math:`\mu_1^{-1} b_1^\perp- \mu_2^{-1}b_2^\perp=K_f`          |
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+
+	- **Normal Component** (:math:`\perp`): Components of fields or fluxes perpendicular to the interface.
+	- **Tangential Component** (:math:`\parallel`): Components of fields or fluxes parallel to the interface.
+	- **Free Surface Charge Density** (:math:`\tau_f`): The free surface charge per unit area on the interface. Occurs due to discontinuities in the normal component of the electric displacement across the interface.
+	- **Free Surface Current Density** (:math:`K_f`): The free surface current per unit area on the interface. Occurs due to discontinuities in the tangential component of the magnetic field across the interface.
+
+
+Derivation of Interface Conditions
```

Comment:
- Regardless of whether there are source terms or not, Maxwell's equations in the quick links section is correct. By introducing a source term, all we are doing is separating the current density (for example) into a primary and a secondary contribution. I see no reason to change it but we could mention that we will discuss sources later and provide the link.
- I feel that the table looks better if, for whatever variable you put in, there is both parallel and perpendicular interface conditions. It means that the reader does not have to infer interface conditions for one variable from another. In regards to the current density, I kept being told how important it was to include interface conditions for DC resistivity here. That's what I did. And this particular simplification for j is only valid for steady-state. A derivation was provided.
- In "solving maxwell's equations", I think you and Doug are kind of missing the point of what I am trying to present. I want to keep it short. I don't want to derive anything which makes the page longer. The intent is for the reader to learn 1) Maxwell's equations can be re-formulated, simplified or approximated for practical use and 2) here are a few examples that you will run into.
- I made the change to the way the interface conditions derivation is listed (using contents instead).
- Added link for N-layered Earth. Great catch!